webapp/webpack.config.js

```diff
@@ -47,6 +49,9 @@ module.exports = {
         'prop-types': 'PropTypes',
         'react-bootstrap': 'ReactBootstrap',
     },
+    plugins: [
+        new NodePolyfillPlugin(),
+    ],
```

Comment:
What's the main reason for this. Were we getting a `process` not defined error?

Reply:
This was done due to the upgrade of webpack version to `5, You can read more about it here: https://webpack.js.org/migrate/5/#upgrade-webpack-to-5, https://webpack.js.org/migrate/5/#upgrade-webpack-to-5, https://stackoverflow.com/a/65556946

Reply:
What error do we get without it though?

Reply:
It's fine but this means we should add it to other plugin projects that will run into this issue too. I wonder if we even need this though. Not sure what Nodejs functions we would need

Reply:
@ayusht2810 What error do we get without adding this?

Reply:
@mickmister Not sure why we are not getting error now. I have reverted the changes and requested QA review again.